benchmark.py

```diff
@@ -176,10 +183,49 @@ def RunBenchmark(args, logfile):
   else:
     print "Instance deletion is skipped due to --no-create_delete."
 
+  ownip = StringIO.StringIO()
+  sh_utils.RunSSHCommand(args.username, args.vm_name,
+                         args=["--command",
+                               ("echo -e \"import benchmark\nprint "
+                                "benchmark.GetOwnIP()\" | python")],
+                         logfile=ownip, zone=args.zone, project=args.project)
+
+  data_store_command = ("python store_data.py --ownip {}"
+                        " --runid {} --envoy_hash {}").format(
+                            ownip.getvalue().strip(),
+                            args.runid, args.envoy_hash)
+
+  if args.create_db_instance:
+    data_store_command = ("{} --create_instance --db_instance_name {}").format(
+        data_store_command, args.db_instance_name)
+  else:
+    data_store_command = "{} --no-create_instance".format(data_store_command)
+
+  if args.create_db:
+    data_store_command = ("{} --create_db --database {}").format(
+        data_store_command, args.database)
+  else:
+    data_store_command = "{} --no-create_db".format(data_store_command)
+
+  if args.delete_db:
+    data_store_command = ("{} --delete_db").format(
+        data_store_command)
+  else:
+    data_store_command = "{} --no-delete_db".format(data_store_command)
```

Comment:
There's a lot of boiler plate for the yes/no case here, can you refactor into a general method for inserting "--no-" in front as needed?